src/library/tactic/simp_lemmas.h

```diff
@@ -137,8 +137,8 @@ simp_lemmas get_simp_lemmas(environment const & env, simp_lemmas_token tk);
 simp_lemmas get_default_simp_lemmas(environment const & env);
 simp_lemmas get_simp_lemmas(environment const & env, name const & tk_name);
 
-simp_lemmas add(type_context_old & ctx, simp_lemmas const & s, name const & id, unsigned priority);
-simp_lemmas add(type_context_old & ctx, simp_lemmas const & s, name const & id, expr const & e, expr const & h, unsigned priority);
+simp_lemmas add(type_context_old & ctx, simp_lemmas const & s, name const & id, bool symm, unsigned priority);
```

Comment:
Should this argument have a default value of `false`?

Reply:
In C++ you can only add default arguments for the last arguments.  Since `priority` has no default value, you can't add one for `symm` either.

You can however define an overload, which you probably should:
```c++
inline simp_lemmas add(type_context_old & ctx, simp_lemmas const & s, name const & id, unsigned priority) {
	return add(ctx, s, id, false, priority);
}
```